Отправил повторно

Dimros12 opened the pull request.

Dimros12 committed:
fix common.css, index.css

Оформил шапку css, добавил в html картинку монстра

Dimros12 committed:
fix index.css

Первый блок

Dimros12 committed:
index.css

Повторный пул реквест

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
+/* Общие данныестраницы ширина, отступ */
 .container {
     max-width: 1180px;
     margin: 110px;
```
Не маржин 110, а margin: 0 auto;

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
 }
 
+/* Общие показатели шапки */
 .header {
+    font-family: "Open Sans", sans-serif;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 19px;
     display: flex;
     align-items: center;
+    text-transform: uppercase;
     margin: 50px 60.05px;
     margin-left: 106px;
 }
```
Всему блоку задавать шрифт и прочие текстовые стили не очень хорошая идея. 
Лучше точечно задать по классу ссылки ей шрифт, размер и т.д.


Но ты можешь основной шрифт сделать на весь документ. 
Основной шрифт - тот который используется везде кроме заголовков

Dimros12 replied:
Это же к шапке относится?

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
     display: flex;
     align-items: center;
+    text-transform: uppercase;
     margin: 50px 60.05px;
```
Дробные значения не используем

Dimros12 replied:
Удалил

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
+
+/* Логотип */
 .header__logo {
     display: flex;
```
Задавать на картинку дисплей флекс не имеет смысла. У тебя он никак не выровняется. Просто лишняя строчка

Dimros12 replied:
Удалил

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
+/* Общие показатели элементов списка */
 .header__nav {
     margin-right: auto;
     text-decoration: none;
```
text-decoration: none; - сделай отдельно на все ссылки, примени по тегу это свойство

Dimros12 replied:
убрал

Yan1993-Dev commented on styles/common.css:
```diff
@@ -3,56 +3,91 @@
 
+/* Отступы между элементами, кроме первого */
 .header__nav-item:not(:last-child) {    
     margin-right: 20.02px;
```
дробное значение округли до 20

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+    display: block;
     background-color: #eee4db;
-    width: 1180px;
+    background-position: center;
```
если у тебя бекграуном не является какое то изображение, то это свойство не нужно. 
У тебя цвет и так по центру

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+    font-weight: 200;
+    font-size: 60px;
+    line-height: 73px;
+    color: rgba(30, 6, 6, 0.05);
```
используй формат hex. Выстави его в фигме

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+.banner__rectangle-list {
+    display: block;
+    font-style: normal;
+    list-style-type: none;
```
можно вынести отдельно для всех ul, li

Dimros12 replied:
Разве все одинаковое не должны объединять?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+    display: block;
+    font-style: normal;
+    list-style-type: none;
+    margin-top: 0;
```
Эта строчка ничего не делает, тут просто значение 0. убери

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+}
+
+.banner__list {
+    position: absolute;
```
зачем списку абсолют задавать? у тебя так не будет нормально отображаться

Dimros12 replied:
Его убрать или заменить? Только чем?

Yan1993-Dev replied:
Не нужно ничем заменять, только убрать абсолют

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+}
+
+.banner__postscript {
+    position: absolute;
```
абсолют тут тоже не нужен

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,112 @@
+.banner__postscript {
+    position: absolute;
+    background: #C29974;
+    max-width: 1180px;
```
у тебя есть контейнер, он задает максимальную ширину, тоже не нужно

Dimros12 replied:
Абсолют убрал и максимальную ширину. Абсолют так жене нужен верно?

Yan1993-Dev replied:
Верно

Yan1993-Dev reviewed (commented):
ок

Dimros12 committed:
index.css

Повторно

Dimros12 committed:
fix index.css

Измененный

Dimros12 committed:
Iindex.css

Сделал первый блок! В телеграмм вопросы остались актуальными

Dimros12 committed:
fix index.css

Products

Dimros12 committed:
fix index.css

Крошки

Dimros12 committed:
fix index.css

начал следующий блок

Dimros12 committed:
fix index.css

Дошел до подвала. Сделал кроме активных кнопок (наведение, клик)

Dimros12 committed:
fix index.css

Сделал первую страницу до подвала. Без активности ссылок и кнопок (наведение, клик)
Проверьте.

Yan1993-Dev commented on index.html:
```diff
@@ -10,7 +10,13 @@
     <link rel="stylesheet" href="./styles/index.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Poppins&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@200;300;400;700&family=Poppins&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;700&family=Open+Sans:wght@300;400;700&family=Poppins&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crimson+Text:ital@1&family=Montserrat:wght@200;300;400;700&family=Open+Sans:wght@200;300;400;700&family=Poppins&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500&display=swap" rel="stylesheet">    
```
А точно нужно так много подключений? Можно же ссылку собрать в одну в гугл фонтс

Yan1993-Dev commented on index.html:
```diff
@@ -36,89 +42,117 @@
-                <h3 class="story__item">Credit</h3>
-                <p class="delivery__text">Some days i feel like i've been working on dlk my entire life and there are other days that i am compleyely shocked that four years have flown by. It's simply magical.</p>
+            <div class="story__item">
+                <img class="story__img" src="./img/girl.jpg" alt="#" width="480" height="559">
```
alt у картинок обязателен к заполнению

Yan1993-Dev commented on index.html:
```diff
@@ -36,89 +42,117 @@
+                <p class="delivery__text-one">Some days i feel like i've been working on dlk my entire life and there are other days that i am compleyely shocked that four years have flown by. It's simply magical.</p>
                 <p class="delivery__text-two">What started as an outlet to deal  with the traumatic loss of my husband, dlk has now become a truly happy place for me and i have you and all of the beautiful designers that we work with to thank!</p>
-                <button class="delivery__btn">LEARN MORE</button>
+                <button class="delivery__btn" style="width: 149px; height: 40px">LEARN MORE</button>
```
зачем инлайн стилизацию писать? Вся стилизация должна быть в файлах css

Yan1993-Dev commented on styles/common.css:
```diff
@@ -1,58 +1,90 @@
 }
 
+/* Параметры названия */
 h1 {
```
Сделай для всех заголовков через запятую

Yan1993-Dev commented on styles/common.css:
```diff
@@ -61,10 +93,10 @@ h1 {
     margin-right: 23.47px;
```
дробные значения не используй, округли до целого

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
     background-color: #eee4db;
-    width: 1180px;
+    display: flex;
     height: 411px;
```
Зачем высота? Можно же контентом растянуть

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+.banner__rectangle-list {
+    display: flex;
+    flex-direction: column;
+    margin-block-start: 0%;
+    padding-inline-start: 0%;
```
Это что за значения?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    border: none;
+    background-color: #1E0606;
+    text-align: center;
+    color: #FFF;
```
используем полный hex формат. #ffffff

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+}
+
+.banner__main {
+    font-family: 'Open Sans', Arial, sans-serif;
```
зачем дополнительный шрифт прописал? его нету в макете

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    justify-content: space-between;
+    /* margin-left: 20px; */
+    margin-top: 30px;
+    padding-left: 0;
```
зачем паддинг 0?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+ 
+}
+
+.products__item {
+    position: absolute;
+    align-items: center;
+}
+
+.products__list-titles {
+    position: absolute;
+    font-family: 'Montserrat', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 26px;
+    color: #FFF;
+    text-align: center;
+    text-decoration: none;
+}
```
зачем тут абсолют? он только ломать будет текст

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+
+.products__item {
+    position: absolute;
+    align-items: center;
```
свойства флекс не работают без display flex

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+
+.products__list-img1 {
+    background-image: url('../img/apparelKids.png');
+    background-blend-mode: multiply;
```
зачем тут это свойство?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    background-blend-mode: multiply;
+}
+
+.products__list-title3 {
+    margin: 119px 100px;
+}
```
к чему тут такие маржины?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    text-transform: uppercase;
+}
+
+.story__delivery {
+    position: absolute;
+    font-family: 'Crimson Text', Arial, sans-serif;
+    font-style: italic;
+    font-weight: 700;
+    font-size: 26px;
+    line-height: 34px;
+    text-decoration: none;
+    margin-top: 7px;
+    background: #d0eefb;
+    width: 326px;
+    height: 55px;
+    padding: 11px;
+    padding-left: 60px;
+    margin-left: -61px;
+}
```
position: absolute; сломает позиционирование
width: 326px;
height: 55px; ширину и высоту из фигмы брать бессмысленно потому что у неё свои значения ширины и высоты. 

    padding-left: 60px;
    margin-left: -61px; они друг друга заменяют в итоге 0 будет, зачем тогда это указывать?

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+}
+
+.story__guarantee {
+    position: absolute;
```
тоже самое, абсолют не нужен здесь

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    align-items: center;
+    padding: 9px 22px;
+    gap: 10px;
+    position: absolute;
```
Зачем тут абсолют. Убери его совсем

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 22px;
+    align-items: center;
+    text-align: center;
```
2 раза align-items: center; и text-align: center; который делает тоже самое, выбери что то одно

Yan1993-Dev commented on styles/index.css:
```diff
@@ -1,17 +1,493 @@
+    text-align: center;
+    text-transform: uppercase;
+    color: #FFF;
+    border: none;
```
border: none; пропиши для всех кнопок

Dimros12 committed:
fix common.css

Подвал

Dimros12 committed:
fix catalog.css

Сделал вторую страницу. Кроме функции наведения и клик

Dimros12 committed:
fix blog.html

blog.html в разоаботке

Dimros12 committed:
fix blog.html

Dimros12 committed:
fix toys.css

Страница toys.css

Dimros12 committed:
Merge branch 'main' of https://github.com/Dimros12/yo-dmitriev-dmitriy-1

Dimros12 committed:
fix contact us

Dimros12 committed:
fix contact us

Доделал страницу без клик и наведения

Dimros12 committed:
fix winmodal.html

оформление

Dimros12 committed:
Merge branch 'main' of https://github.com/Dimros12/yo-dmitriev-dmitriy-1
